### PR TITLE
Merchant dashboard show page

### PR DESCRIPTION
**Pull request by lrs8810**
- (#14 ) Merchant dashboard show page which includes the name and address of the merchant the merchant employee/admin works for

### CODE REVIEW

File: app/controllers/merchant/dashboard_controller.rb
Comment (lrs8810) [inline app/controllers/merchant/dashboard_controller.rb:4]: Happy to change this before merging to a #show action instead of an #index action due to this being a merchant dashboard show page. I was holding off on updating the action to avoid too many merge conflicts.

File: spec/features/users/merchant_dash_spec.rb
Comment (lrs8810) [inline spec/features/users/merchant_dash_spec.rb:6]: These tests will be on the same spec as #88.

File: app/controllers/merchant/dashboard_controller.rb
Comment (freeheeling) [reply app/controllers/merchant/dashboard_controller.rb:4]: Show seems more appropriate.

File: spec/features/users/merchant_dash_spec.rb
Comment (freeheeling) [inline spec/features/users/merchant_dash_spec.rb:2]: The merge should should include the View Items link referenced in user story (#12).

File: app/views/merchant/dashboard/index.html.erb
Comment (freeheeling) [inline app/views/merchant/dashboard/index.html.erb:1]: I think a heading, such as 'Employer', 'Merchant' or similar, would be helpful to specify for these details.

File: app/controllers/merchant/dashboard_controller.rb
Comment (cjkelling) [reply app/controllers/merchant/dashboard_controller.rb:4]: This is for user story 30 correct? It says that the dashboard should show a list of all pending orders. A list of orders would indicate an index.

Comment (lrs8810) [reply app/controllers/merchant/dashboard_controller.rb:4]: @cjkelling this is user story 29 (#14 )

File: app/views/merchant/dashboard/index.html.erb
Comment (lrs8810) [reply app/views/merchant/dashboard/index.html.erb:1]: @freeheeling when you say, "heading" do you mean changing the id #merchant_details to #employer?

File: app/controllers/merchant/dashboard_controller.rb
Comment (cjkelling) [reply app/controllers/merchant/dashboard_controller.rb:4]: Interesting. User story 29 and user story 30 both route to '/merchant'. #29 says the page needs to display the merchant information, and #30 says it should list all pending orders. It appears that the same page will need to render both sets of information. Being such, I think you can argue calling it either show or index. Whatever the group wants to do.

Comment (cjkelling) [reply app/controllers/merchant/dashboard_controller.rb:4]: And I see I used the number indicators incorrectly. Story 29 is #14 and story 30 is #13.

**Review by cjkelling — APPROVED** [review]
These files look good. Making the user have an optional relationship with a merchant allows us to create many different types of users. Overall this code is well done.

**Pull request merged**
